### PR TITLE
Moved scripts from plugin repository

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
 # user-activity-monitoring-scripts
 Scripts for the User Activity Monitoring plugin
+
+## User guide
+In order to generate such reports across multiple masters, we have prepared two scripts:
```
Comment (davidcurrie) [inline README.md]: Missing context of `such reports`. Also, there appear to be three scripts, not two.

**Comment by fiunchinho** [comment]
@davidcurrie asked a very good question: do we want `install-user-activity-monitoring-plugin.sh` and `standard-master-plug-catalog.json` copied across? do they still have a use for air gap customers until the plugin is included in the WAR?

**Comment by davidcurrie** [comment]
@fiunchinho - @jphartley replied in Slack to indicate that they are not needed

**Comment by fiunchinho** [comment]
Fixed

**Comment by cyrille-leclerc** [comment]
Thanks @fiunchinho

**Pull request merged**